src/Build.UnitTests/Evaluation/Expander_Tests.cs

```diff
@@ -3518,5 +3519,23 @@ public void PropertyFunctionEnsureTrailingSlash()
 
             Assert.Equal(path + Path.DirectorySeparatorChar, result);
         }
+
+        [Fact]
+        public void PropertyFunctionWithNewLines()
+        {
+            const string propertyFunction = @"$(SomeProperty
+.Substring(0, 10)
+.Substring(0, 5))";
```

Comment:
Yes the unit test covers all the changed lines.  The first changed line trims the first property/method parsed and the second and third trims the remainder for methods with and without arguments respectively.